src/objects/conf/production.py

```diff
@@ -4,6 +4,7 @@
 
 from .base import *  # noqa isort:skip
 
+os.environ.setdefault("RELEASE", API_VERSION)
```

Comment:
`API_VERSION` is the version of the API endpoint itself and we want to show the version of the application in the admin. This should be the `RELEASE` setting defined in `open-api-framework` https://github.com/maykinmedia/open-api-framework/blob/main/open_api_framework/conf/base.py#L640. I'm not sure why it isn't picked up here though, maybe it's not being set in the Docker image?